packages/console/src/components/Topbar/index.tsx

```diff
@@ -14,17 +14,19 @@ import * as styles from './index.module.scss';
 
 type Props = {
   className?: string;
+  hasTenantSelector?: false;
+  hasTitle?: false;
 };
 
-function Topbar({ className }: Props) {
+function Topbar({ className, hasTenantSelector, hasTitle }: Props) {
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
   const LogtoLogo = isCloud ? CloudLogo : Logo;
 
   return (
     <div className={classNames(styles.topbar, className)}>
       <LogtoLogo className={styles.logo} />
-      {isCloud && <TenantSelector />}
-      {!isCloud && (
+      {isCloud && hasTenantSelector !== false && <TenantSelector />}
```

Comment:
what if `hasTenantSelector` and `hasTitle` is undefined?

Reply:
Then they will still be rendered. Only explicitly set it to `false` can it be hidden

Reply:
how about replacing it with `hideTenantSelector`?

Reply:
It should also be fine. Then we'll have to disable the mandate `has|is` prefix for boolean prop lint rule for this one

Reply:
Updated to use `boolean` type instead